Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
       actionpack (>= 5.0)
       activesupport (>= 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.573.0)
-    aws-sdk-core (3.130.0)
+    aws-partitions (1.632.0)
+    aws-sdk-core (3.150.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-kms (1.55.0)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-kms (1.58.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.113.0)
+    aws-sdk-s3 (1.114.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.4)
-    aws-sigv4 (1.4.0)
+    aws-sigv4 (1.5.1)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.0 → 1.5.1

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sigv4), [gem-diff](https://my.diffend.io/gems/aws-sigv4/1.4.0/1.5.1)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.4.0...1.5.1).

These are the individual commits:
- (623d9e9) [Version 1.4.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/623d9e91f278caffe85f9b7e8ace1a1cfa46de03)
- (a8e3c37) [defualt -> default](https://github.com/aws/aws-sdk-ruby/commit/a8e3c3798b48ff549d12eb678ad4f2426d074ecf)
- (ffc9307) [Merge pull request #40 from kml/master](https://github.com/aws/aws-sdk-ruby/commit/ffc930717d27e7e6a632c547494e6e356d5a0d1e)
- (683488d) [IT everywhere. ConfiguraITon -> ConfiguraTIon](https://github.com/aws/aws-sdk-ruby/commit/683488d3daf1dd61013f88d714b34904c25704f0)
- (970a919) [Merge pull request #42 from kml/master](https://github.com/aws/aws-sdk-ruby/commit/970a9197a4a7078b425071f9bd72cfcd23a2f640)
- (03b38c9) [Version 1.5.0 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/03b38c9b441baece5f2c34d8e713edcfe2feb3e5)
- (5f6d901) [Version 1.5.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/5f6d9019251c2e497188037662e93a78b69ace2d)


</details>